xDrip/Model/User/Settings/AlertSettings.swift

```diff
@@ -35,6 +35,32 @@ final class AlertSettings: Object {
         }
     }
     
+    func getSound(for type: AlertEventType) -> CustomSound {
+        var sound = CustomSound.tritone
```

Comment:
Let's create a `static var default: CustomSound` in `CustomSound` to provide a default sound.